BoxOffice/View/MovieDetailView.swift

```diff
@@ -0,0 +1,448 @@
+    
+    private let nationsContentLabel: UILabel = {
+        let label = UILabel()
+        
+        label.font = .systemFont(ofSize: 18)
+        label.setContentHuggingPriority(.init(1), for: .horizontal)
+        return label
+    }()
+    
+    private let genresStackView: UIStackView = {
+        let stackView = UIStackView()
+        
+        stackView.axis = .horizontal
+        return stackView
+    }()
+    
+    private let genresLabel: UILabel = {
+        let label = UILabel()
+        
+        label.font = .boldSystemFont(ofSize: 18)
+        label.text = "장르"
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private let genresContentLabel: UILabel = {
+        let label = UILabel()
+        
+        label.font = .systemFont(ofSize: 18)
+        label.setContentHuggingPriority(.init(1), for: .horizontal)
+        return label
+    }()
+    
+    private let movieActorsStackView: UIStackView = {
+        let stackView = UIStackView()
+        
+        stackView.axis = .horizontal
+        return stackView
+    }()
+    
+    private let movieActorsLabel: UILabel = {
+        let label = UILabel()
+        
+        label.font = .boldSystemFont(ofSize: 18)
+        label.text = "배우"
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private let movieActorsContentLabel: UILabel = {
+        let label = UILabel()
+        
+        label.font = .systemFont(ofSize: 18)
+        label.numberOfLines = 0
+        label.setContentHuggingPriority(.init(1), for: .horizontal)
+        return label
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setUpLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setUpContents(_ movieDetailInformationDTO: MovieDetailInformationDTO) {
+        DispatchQueue.main.async {
+            self.directorContentLabel.text = movieDetailInformationDTO.directors.joined(separator: ", ")
+            self.productionYearContentLabel.text = movieDetailInformationDTO.productYear
+            self.openDateContentLabel.text = movieDetailInformationDTO.openDate
+            self.showTimeContentLabel.text = movieDetailInformationDTO.showTime
+            self.watchGradeContentLabel.text = movieDetailInformationDTO.watchGrade
+            self.nationsContentLabel.text = movieDetailInformationDTO.nations.joined(separator: ",")
+            self.genresContentLabel.text = movieDetailInformationDTO.genres.joined(separator: ", ")
+            self.movieActorsContentLabel.text = movieDetailInformationDTO.movieActors.joined(separator: ", ")
+            
+            if movieDetailInformationDTO.isMovieActorsEmpty() {
+                self.movieActorsStackView.isHidden = true
+            }
```

Comment:
```suggestion
            self.movieActorsStackView.isHidden = movieDetailInformationDTO.isMovieActorsEmpty()
```

if문의 조건에 따라 Bool 변수의 값이 달라진다면 if문의 조건 자체를 할당하는 것도 좋겠습니다.

Reply:
넵! 변경진행했습니다. 감사합니다.
(https://github.com/yagom-academy/ios-box-office/pull/98/commits/846dafcd407b5e8bdd956afa47f5149506a0e53a)